---
layout: default
title: "Robert Stones"
---

Robert started working at the BSU in January 2015 as a Bioinformatics Experimental Officer in the Faculty of Medical Sciences.

Robert completed his Masters degree in Bioinformatics, with Distinction in October 2000 at York University, following an undergraduate degree in Molecular Biology and Biochemistry at Durham University.

Previously Robert worked as a lead support bioinformatician for FERA, a UK government science agency based near York, UK. Supporting laboratory based scientists in work areas including next generation DNA sequencing technologies, metabolomics (NMR) and proteomics (MS) technologies. Robert has experience of collaborating on a number of projects with international partners from both the USA and the EU.

Robert also has a keen interest in computer science and software engineering, and as well as utilizing existing analytical tools/methods; has interests in building user interfaces for data storage/analysis and data visualization, in collaboration with colleagues for laboratory based scientists.

Robert is based at the BSU (2nd Floor Leech Building) and can be contacted at <a href="mailto:robert.stones@ncl.ac.uk">robert.stones@ncl.ac.uk</a>.
<h2><strong>Publications</strong></h2>
Cao G, Allard M, Strain E, <strong>Stones R</strong>, Zhao S, Brown E, Meng J. Genetic diversity of Salmonella pathogenicity islands SPI-5 and SPI-6 in <em>Salmonella Newport</em>. <strong><em>Foodborne Pathog Dis.</em></strong> 2014 Oct;11(10):798-807. 2014 Sep 4.

Parkinson N, Bryant R, Bew J, Conyers C, <strong>Stones R</strong>, Alcock M, Elphinstone J. Application of variable-number tandem-repeat typing to discriminate <em>Ralstonia solanacearum</em> strains associated with English watercourses and disease outbreaks. <strong><em>Appl Environ Microbiol</em>.</strong> 2013 Oct;79(19):6016-22. 2013 Jul 26.

Cao G, Meng J, Strain E, <strong>Stones R</strong>, Pettengill J, Zhao S, McDermott P, Brown E and Allard MW. (2013) Phylogenetics and differentiation of <em>Salmonella Newpor</em>t lineages by whole genome sequencing. <strong><em>PLoS One</em></strong>.;8(2).

Allard MW, Luo Y, Strain E, Pettengill J, Timme R, Wang C, Li C, Keys CE, Zheng J, <strong>Stones R</strong>, Wilson MR, Musser SM, Brown EW (2013) On the evolutionary history, population genetics and diversity among isolates of <em>Salmonella</em> <em>Enteritidis</em> PFGE pattern JEGX01.0004. <strong><em>PLoS One</em></strong>.;8(1)

Timme R, Allard MW, Luo Y, Strain E, Pettengill J, Wang C, Cong L, Keys CE, Zheng J, <strong>Stones</strong><strong> R</strong>, Wilson M, Musser SM, and Eric Brown EW ( 2012)   Draft Genome Sequence of 21 <em>Salmonella</em> <em>Enteritidis</em> strains. <strong><em>Journal of Bacteriology</em></strong> 2012 Nov;194(21):5994-5.

Allard MW, Luo Y, Strain E, Li C, Keys CE, Son I, <strong>Stones R</strong>, Musser SM, Brown EW (2012) High resolution clustering of <em>Salmonella enterica serovar Montevideo</em> strains using a Next-generation Sequencing approach. <strong><em>BMC Genomics</em></strong> Jan 19; 13:32.

Lienau EK, Strain E, Wang C, Zheng J, Ottesen AR, Keys CE,Hammack T<sup>1</sup>,Musser SM,Brown EW,Allard MW,Cao GJ,Meng JH and <strong>Stones R</strong> (2011) Identification of a Salmonellosis Outbreak by Means of Molecular Sequencing. <strong><em>New England Journal of Medicine</em></strong> 364(10), 981-982.

Bell RL,Gonzalez-Escalona N,<strong>Stones R</strong>, Brown EW (2011) Phylogenetic evaluation of the 'Typhimurium' complex of Salmonella strains using a seven-gene multi-locus sequence analysis <strong><em>Infection Genetics and Evolution</em></strong>. 11(1), 83-91.

Macarthur R, Murray AW, Allnutt TR, Deppe C, Hird HJ, Kerins GM, Blackburn J, Brown J, <strong>Stones R</strong>, Hugo S. Model for tuning GMO detection in seed and grain. <strong><em>Nat Biotechnol</em></strong>. 2007 Feb;25(2):169-70.

Booth, C., Boonham, N., <strong>Stones, R.</strong>, Iturriza-Gomara, M., Gray, J. and Cook, N. (2003). A PCR microarray based approach for identification of rotavirus strains<em>. <strong>FEMS Microbiology Letters</strong></em> 222 1, 279.

M.Crosier, L. Viggiano, J. Guy, D. Misceo, <strong>Stones, R.</strong>, W. Wei, T. Hearn, M. Rocchi, and M. Jackson. (2002). Human paralogues of KIAA0187 created through independent pericentromeric-directed and chromosome-specific duplication mechanisms. <strong><em>Genome Research</em> </strong>Jan; 12(1):67-80.

&nbsp;
